src/management/OrganizationsManager.js

```diff
@@ -289,7 +290,9 @@ OrganizationsManager.prototype.getEnabledConnections = function(params, callback
  *   console.log(enabled_connection);
  * });
  *
- * @param   {String}    [organization_id]   Organization ID
+ * @param   {Object}    params        Organization parameters.
+ * @param   {String}    params.id     Organization ID.
+ * @param   {String}    params.connection_id     Connection ID.
```

Comment:
The generated docs is wrong:

![image](https://user-images.githubusercontent.com/2146903/135079567-a6f9686d-a20a-4ff8-888f-3f6ddf0b9471.png)

As you can see it looks like it expects an optional `organization_id`, while it expects a required `{id: ''}`